server-side/controllers/user.controller.js

```diff
@@ -28,8 +29,42 @@ export const getUserById = async (req, res) => {
   }
 };
 
-export const addUser = async (req, res) => {
+let transporter = nodemailer.createTransport({
```

Comment:
you shouldn't send an email from here, you should call to email service that Malki Sinowitz and Baty Finklshtein created, ask them about details

Reply:
why did you remove all the calls for sending emails that you had? didn't you need them already?